ign_ros2_control/src/ign_system.cpp

```diff
@@ -627,6 +637,25 @@ hardware_interface::return_type IgnitionSystem::write(
           {this->dataPtr->joints_[i].joint_effort_cmd});
       }
     }
+
+    // Fallback case is a velocity command of zero
```

Comment:
I think this is what prevented the unactuated joints from drooping with gravity.